docker-compose.yaml

```diff
@@ -0,0 +1,19 @@
+version: "3"
+
+services:
+  postgresdb:
+    image: postgres:11.5
```

Comment:
Image 11.5 is already a bit older. Can we just use `postgres:latest` here. I think we also should change it in the other application